app/controllers/attachments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class AttachmentsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
O authenticate_user pode estar no application controller, ou em um base controller pra garantir que n vai ter repetição de codigo redundante toda vez que um novo controller for criado.

Reply:
Subi um pr corrigido o comments que estava com isso também e adicionado ele ao  ApplicationController

Reply:
Perfeito, como isso foi adicionado lá ao application, não se faz necessário ter ele repetido aqui visto que o AttachmentsController herda do ApplicationController